SpatialGDK/Documentation/features.md

```diff
@@ -32,13 +32,13 @@ The SpatialOS Unreal GDK also supports:
 * Replicated Blueprint events
 * RPC validation
 
-## Static subobject replication
+## Subobject replication
 
-The SpatialOS Unreal GDK currently supports only one instance of each static subobject type when replicating objects owned by an Actor. This includes components. For example, multiple Actors can have a `FooComponent` but no Actor can have two `FooComponent`s.
+The GDK currently supports only one instance of each static subobject type when replicating objects owned by an Actor. This includes components. For example, multiple Actors can have a `FooComponent` but no Actor can have two `FooComponent`s.
```

Comment:
This includes components.
->
This includes and Actor's components.

(Note from editor - in the para above we are talking about SpatialOS components, so we should be absolutely clear - I know you didn't edit this, Giray but I just spotted the potential lack of clarity.)